emcc.py

```diff
@@ -1127,7 +1127,13 @@ def version_string():
   return f'emcc (Emscripten gcc/clang-like replacement + linker emulating GNU ld) {utils.EMSCRIPTEN_VERSION}{revision_suffix}'
 
 
-def parse_args(newargs):
+def parse_args(newargs):  # noqa: C901, PLR0912, PLR0915
+  """
```

Comment:
I think docstrings like this are always supposed to start with the first line on the same like as the opening tipple quote.

See https://peps.python.org/pep-0257/